OrbitClientGgp/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+project(OrbitClientGgp)
+
+add_executable(OrbitClientGgp main.cpp)
+
+target_compile_options(OrbitClientGgp PRIVATE
+    ${STRICT_COMPILE_FLAGS})
+
+target_include_directories(OrbitClientGgp PRIVATE 
+    ${CMAKE_CURRENT_LIST_DIR})
+
+target_sources(OrbitClientGgp PRIVATE
+    ClientGgp.cpp
+    ClientGgp.h
+    main.cpp)
+
+target_link_libraries(OrbitClientGgp PUBLIC 
+    OrbitBase
+    OrbitCaptureClient
+    OrbitProtos)
+
+#strip_symbols(OrbitClientGgp)
```

Comment:
Can you remove that line if not needed? Is there a reason why you commented that?

Reply:
Done. I saw it was in the OrbitService so I added and commented out here in case I needed as well. I don't think I do in the end, so I removed it